bundles/core/src/test/java/com/adobe/cq/commerce/core/components/internal/models/v1/productlist/ProductListImplTest.java

```diff
@@ -125,8 +125,7 @@ public void setUp() throws Exception {
 
         Utils.setupHttpResponse("graphql/magento-graphql-introspection-result.json", httpClient, HttpStatus.SC_OK, "{__type");
         Utils.setupHttpResponse("graphql/magento-graphql-attributes-result.json", httpClient, HttpStatus.SC_OK, "{customAttributeMetadata");
-        Utils.setupHttpResponse("graphql/magento-graphql-category-result.json", httpClient, HttpStatus.SC_OK, "{category");
```

Comment:
I think `magento-graphql-category-result.json` is no longer needed, you can use your new file in in the `setUp()` method to do `category = Utils.getQueryFromResource("graphql/magento-graphql-search-result-with-category.json").getCategory();`